README.md

```diff
@@ -1,48 +0,0 @@
-# ZIO Prelude
```

Comment:
README.md should still be part of the repository, right?

Reply:
Oh, so GitHub Action will change code in `master`, correct?

Reply:
Yes